src/sttn/data/restaurants.py

```diff
@@ -0,0 +1,90 @@
+import pandas as pd
+import geopandas as gpd
+import numpy as np
+
+from sttn import network
+from .data_provider import DataProvider
+
+class RestaurantInspectionDataProvider(DataProvider):
+    """New York City Restaurant Inspections data provider, builds a network where nodes represent restaurants
+     and every edge represents an inspection where the origin and destination point to the node
+     where the inspection happened."""
+  
+    @staticmethod
+    def get_data(self, from_date, to_date) -> network.SpatioTemporalNetwork:
```

Comment:
It's good practice to add the typing information for function arguments: `from_date: str`